content/news/2020-10-04-wallpaper-contest.md

```diff
@@ -0,0 +1,39 @@
++++
+type = "news"
+title = "Call for Wallpapers"
+author = "bitigchi"
+date = "2020-10-08 13:00:00+03:00"
+tags = ["wallpapers", "contest"]
++++
+
+One of the most common feedback that we receive is that Haiku needs some nice colourful wallpapers shipping with its releases. We definitely want to improve the current situation. And we need your help.
+
+Haiku currently ships with only one (1) wallpaper, and it's not a complete wallpaper per se, it's just the Haiku logo positioned at fixed coordinates on the screen. To be fair, there is a reason we only use the Haiku logo as the default wallpaper; we want Haiku to be usable on systems that have memory as low as 256 MB, therefore each byte that we can spare counts.
```

Comment:
```suggestion
Haiku currently ships with only one (1) wallpaper, and it's not a complete wallpaper per se, it's just the Haiku logo positioned at fixed coordinates on the screen. To be fair, there is a reason we only use the Haiku logo as the default wallpaper; we want Haiku to be usable on systems that have memory as low as 256 MB, therefore each byte that we can spare counts.
It's also works relatively well with different screen resolutions.
```

Reply:
Web+ won't let me edit the above... It's of course:

> It also works relatively well with different screen resolutions.

I can imagine that this is actually the main reason. I'm not sure memory is even an issue here; doesn't it always take e.g. 1920*1080*32bit, no matter if it's a photo or a one-colour background?